bricks/flutter_clean_base/__brick__/{{project_name.snakeCase()}}/lib/data/source/remote/firebase/auth/firebase_auth_source_impl.dart

```diff
@@ -70,7 +68,7 @@ class FirebaseAuthSourceImpl extends FirebaseAuthSource {
         return const DataResponse.unauthorized();
       }
     } on FirebaseAuthException catch (e) {
-      return DataResponse.firebaseError(e.code);
+      return DataResponse.undefinedError(e);
```

Comment:
@cozvtieg9,  I removed the firebaseError. There is no way to extend the freezed class